src/client/components/card/components/Card.jsx

```diff
@@ -1,14 +1,86 @@
+import React, { PropTypes } from 'react'
 import { Card } from 'material-ui/Card'
 import styled from 'styled-components'
 
+const SIZE = { small: 'small', medium: 'medium', large: 'large' }
```

Comment:
I think just using an enum propType is good enough here. That way you don't have to manually check if the size exists or not and it's a little less verbose.